cv32e40p/regress/cv32e40pv2_benchmarks.yaml

```diff
@@ -16,21 +16,45 @@ builds:
   uvmt_cv32e40p:
     cmd: make comp comp_corev-dv
     dir: cv32e40p/sim/uvmt
-    cfg: pulp
+    cfg: pulp_fpu
 
+# ====================================================================================
 # List of tests
 tests:
+  dhrystone:
+    build: uvmt_cv32e40p
+    description: dhrystone
+    dir: cv32e40p/sim/uvmt
+    cmd: make test TEST=dhrystone CFG_PLUSARGS="+UVM_TIMEOUT=1000000"
+    num: 1
+
+  fibonacci:
```

Comment:
Does this `fibonacci` test add to our total coverage?  It was "found" in a directory of old RIS5CY test-programs and I copied it over.  Maybe it is time to retire it.  (Of course, it doesn't hurt to keep it...)

Maybe if we increased the length of the calculated sequence and enabled random interrupts.

Reply:
There are tests for verification and tests for benchmarking.
The second category is not there for any verification purpose.
But I agree I never used that fibonacci test in the past.
And we are going to use Embench additionally to Dhrystone and Coremark for performance benchmarking.